stanford/roles/cluster/tasks/load_balancer.yml

```diff
@@ -0,0 +1,22 @@
+- name: Lookup/create load balancer
```

Comment:
You shouldn't need to do this.

ELB creation was already conditional;
it was only created if you had configured it via `CLUSTER_ELBS[CLUSTER_NAME]`.

No ELB will be created otherwise.